database/init_db.go

```diff
@@ -19,8 +19,8 @@ type flags struct {
 }
 
 var defaultValues = &flags{
-	DatabaseName:     "postgresdb",
-	DatabaseUser:     "ps_user",
+	DatabaseName:     "kubearchive",
+	DatabaseUser:     "kubearchive",
```

Comment:
These changes won't apply anymore after https://github.com/kubearchive/kubearchive/pull/346/files PR

Reply:
Yes, I am aware. If mine goes first then they are needed, if Andrea's go first this file will disappear when I rebase.